src/table/hooks/use-focus-listener.ts

```diff
@@ -1,10 +1,12 @@
 import { stardust } from '@nebula.js/stardust';
-import { useEffect } from 'react';
+import React, { useEffect } from 'react';
```

Comment:
Why React is needed in this file?

Reply:
... I thought I'm adding it because of line 6 but seems it works without importing React anyway. I'll remove it